util/memory-tracker/src/process.rs

```diff
@@ -110,11 +110,11 @@ pub struct Memory {
     // Size of physical memory being used
     resident: u64,
     // Number of shared pages
-    shared: u64,
+    _shared: u64,
```

Comment:
Why adding the prefix `_`?

Reply:
Higher versions of rustc will report a warning that this data is not used.
I'm not sure if I'll need this data in the future, so I'll add `_` to disable the warning for now